var/spack/repos/builtin/packages/py-awscrt/package.py

```diff
@@ -16,4 +16,11 @@ class PyAwscrt(PythonPackage):
 
     version("0.16.16", sha256="13075df2c1d7942fe22327b6483274517ee0f6ae765c4e6b6ae9ef5b4c43a827")
 
+    depends_on("cmake", type=("build"))
```

Comment:
```suggestion
    depends_on("cmake@3.1:", type=("build"))
```
Based on `cmake_minimum_required`

Reply:
Thanks!